### PR TITLE
#153756400 Convert from grunt to Gulp taskrunner

### PR DESCRIPTION
**Pull request by kevinejiro**
### What does this PR do?
Converts from grunt to gulp task runner

### Description of Task to be completed?
Grunt is old; with Gulp, the syntax is much simpler and it uses pipe streams which makes it inherently faster.

### How should this be manually tested?
Run `npm install` (To Install all dependencies )
Run `npm start` (To start gulp)

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
153756400

### Screenshots (if appropriate)
screen shot 2018-01-03 at 5 22 26 pm

### Questions:
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,25 +13,23 @@
     "npm": "3.10.10"
   },
   "scripts": {
-    "start": "node node_modules/grunt-cli/bin/grunt",
-    "test": "node node_modules/grunt-cli/bin/grunt test",
-    "postinstall": "node node_modules/grunt-cli/bin/grunt install",
+    "start": "node node_modules/gulp/bin/gulp build && node node_modules/gulp/bin/gulp",
```
Comment (temilaj) [inline package.json]: why do we have to run gulp twice?

**Review by temilaj — CHANGES_REQUESTED** [review]
made a comment. please review

File: package.json (hunk 1 above)
Comment (kevinejiro) [reply package.json]: I did that because we might have to refactor our code to es6 which might give problems on heroku.

**Review by temilaj — CHANGES_REQUESTED** [review]
Both your commits seem to be doing the same thing. Please squash them into a single commit. Or have separate commits for the different tasks completed

**Pull request closed without merging**